convert2rhel/systeminfo.py

```diff
@@ -138,9 +139,10 @@ def _get_gpg_key_fingerprints(self):
     def _get_pkg_blacklist(self):
         return self._get_cfg_opt("pkg_blacklist").split()
 
-    def _generate_rpm_va(self):
-        """Verify the installed files on the system with the rpm metadata
-        for debug and support purposes."""
+    def _generate_rpm_va(self, log_filename=DEFAULT_RPM_VA_LOG_FILENAME):
```

Comment:
Another test to check whether `log_filename` param works as expected

Reply:
I believe there's no need to test the `log_filename`, because we accept any value, any value you put that will be used, even blank. If you put blank value, will used a const, if you pass any value, will be used.. so, no reason to test this argument since we accept any value.

Reply:
It should be tested, though I made a mistake. The tests seem to cover both using the parameter and the default value as well.